kms_log/main.tf

```diff
@@ -558,4 +593,469 @@ resource "aws_cloudwatch_metric_alarm" "dead_letter" {
     alarm_actions = [
         "${var.sns_topic_dead_letter_arn}"
     ]
+}
+
+#lambda functions
+resource "aws_lambda_function" "cloudtrail_processor" {
+    count = "${var.kmslogging_service_enabled}"
+    s3_bucket = "${data.aws_s3_bucket.lambda.id}"
+    s3_key = "circleci/identity-lambda-functions/${var.lambda_identity_lambda_functions_gitrev}.zip"
+
+    lifecycle {
+        ignore_changes = ["s3_key", "last_modified"]
+    }
+
+    function_name = "${var.env_name}-kmslog-ct-processor"
+    description = "18F/identity-lambda-functions: KMS CT Log Processor"
+    role = "${aws_iam_role.cloudtrail_processor.arn}"
+    handler = "main.Functions::KMSCloudTrailHandler.process"  #TODO Verify
+    runtime = "ruby2.5"
+    timeout = 30 # seconds
+
+    environment {
+        variables = {
+            DEBUG = "${var.lambda_audit_github_debug ? "1" : ""}"
```

Comment:
I can't see where this is defined. Maybe it could be its own variable specific to this module?

Reply:
I have updated the variable name and added the variable to the variables file.